regimes/pt/pt.go

```diff
@@ -16,6 +17,13 @@ func init() {
 	tax.RegisterRegime(New())
 }
 
+// Custom keys used typically in meta information
+const (
+	KeyTaxCode          cbc.Key = "at-tax-code"
+	KeyTaxRegion        cbc.Key = "at-tax-country-region"
+	KeyTaxExemptionCode cbc.Key = "at-tax-exemption-code"
```

Comment:
These correspond to the `TaxCode`, `TaxCountryRegion` and `TaxExemptionCode` fields present both in the SAFT and [webservice communication](https://info.portaldasfinancas.gov.pt/pt/apoio_contribuinte/Faturacao/Fatcorews/Documents/Comunicacao_dos_elementos_dos_documentos_de_faturacao.pdf) with the AT.